monitoring/definitions/codeintel_autoindexing.go

```diff
@@ -10,15 +10,27 @@ func CodeIntelAutoIndexing() *monitoring.Dashboard {
 		Name:        "codeintel-autoindexing",
 		Title:       "Code Intelligence > Autoindexing",
 		Description: "The service at `internal/codeintel/autoindexing`.",
-		Variables:   []monitoring.ContainerVariable{},
+		Variables: []monitoring.ContainerVariable{
+			{
+				Label: "Source",
+				Name:  "source",
+				OptionsLabelValues: monitoring.ContainerVariableOptionsLabelValues{
+					Query:         "src_codeintel_autoindexing_total{}",
+					LabelName:     "app",
+					ExampleOption: "frontend",
+				},
+				WildcardAllValue: true,
+				Multi:            false,
+			},
+		},
 		Groups: []monitoring.Group{
-			shared.CodeIntelligence.NewAutoindexingSummaryGroup(""),
-			shared.CodeIntelligence.NewAutoindexingServiceGroup(""),
-			shared.CodeIntelligence.NewAutoindexingGraphQLTransportGroup(""),
-			shared.CodeIntelligence.NewAutoindexingStoreGroup(""),
-			shared.CodeIntelligence.NewAutoindexingBackgroundJobGroup(""),
-			shared.CodeIntelligence.NewAutoindexingInferenceServiceGroup(""),
-			shared.CodeIntelligence.NewLuasandboxServiceGroup(""),
+			shared.CodeIntelligence.NewAutoindexingSummaryGroup("$source"),
```

Comment:
Can you try changing this to use the `regex` processor thingamajig:

```suggestion
			shared.CodeIntelligence.NewAutoindexingSummaryGroup("${source:regex}"),
```

there's another query that does this so I think the dashboard should work, and I confirmed the replacement works correctly in this case

Reply:
Look good to you? [sourcegraph/sourcegraph@`be3d6e3` (#43865)](https://github.com/sourcegraph/sourcegraph/pull/43865/commits/be3d6e389fa862be769b43308c053a9422d8cfe5)